.github/workflows/ci.yml

```diff
@@ -20,39 +20,59 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```

Comment:
I kept getting inexplicable errors with MXNet tests (see explanation in #615) when I tried to use a later versions of Ubuntu. Just in case you end up running into issues with this like I did.

Reply:
Ok, I remember. Let's see if it works now. We also could see if we can upgrade `mxnet` as part of this PR to fix the problem.